### ISSUE TITLE
Layout problems on iOS - Apple rejection - still on 2.18.6

### ISSUE THREAD

**Issue by albertoiglesias2001** [AlejandroRuiz/PayPal.Forms#88]
I'm developing on iPhone 8 (iOS 13)  :-(

Updated right now to test this new version but still the same layout problem, now have another one on PAY WITH CARD with icon behide

![60f9fd8a-b9e6-4aa2-abab-84218ea57fc3](https://user-images.githubusercontent.com/18347714/66782957-ceb2aa80-eece-11e9-999b-2c8a102a1176.jpg)

**Comment by AlejandroRuiz** [comment]
As im mentioned that is not supported anymore by Paypal so we don't have a way to direct modify the UI appearance, but let me check if I can do something for you

**Comment by albertoiglesias2001** [comment]
OK.  If I can help with something, just let me now, I have some libraries created by me too

**Comment by AlejandroRuiz** [comment]
@albertoiglesias2001 I paused the project since I didn't find time to complete it but im still planing to deliver it with no ETA

**Comment by stavroaudi** [comment]
@AlejandroRuiz please allow @albertoiglesias2001 to help as I would like to replace Paypal.Forms with Braintree.Forms on my app. Thank you!

**Comment by swansca79** [comment]
The Braintree mobile packages are behind 2 years on updates, its that really a suitable replacement for this project?

**Comment by AlejandroRuiz** [comment]
@swansca79 the Braintree project doesn't have any ETA as I didn't find time to complete it in any case that you want to create your own implementation you can go directly to the Braintree docs and use it https://www.braintreepayments.com that library its a quite big with a totally different configuration and behavior from what Paypal Mobile SDK used to have, so you're welcome to contribute on the project if you want

**Comment by swansca79** [comment]
Right I understand, I was really referencing all these projects by [Naxam](https://github.com/NAXAM/braintreedropin-android-binding).  They have all the bindings, but they are just old and they didn't check in the binding projects.  There are way too many projects for me to get these all bound, and since I really just need PayPal, your project seemed like a better option.

**Comment by AlejandroRuiz** [comment]
Yes as I mentioned that new library its quite big so it have a lot of features, and thats why I recommend to you to create your bindings and your cross implementation based on your needs that will allow you to have a faster implementation, Braintree is releasing new versions of their sdks often so you can get those sdks direct here and you can create your own bindings and implementations based on their docs https://developers.braintreepayments.com/

**Comment by swansca79** [comment]
I was able to get those Naxam packages to work.  They're old, but they provide enough functionality for PayPal vaulting, which is what I'm using in my app.  The drop-in UI worked too, so if you need full checkout that should work too.
